lib/kalevala/character/conn.ex

```diff
@@ -160,6 +160,7 @@ defmodule Kalevala.Character.Conn do
     conn.session
     |> Map.put(:character, Private.character(conn))
     |> Map.merge(conn.assigns)
+    |> Map.merge(conn.flash)
```

Comment:
I think we want `flash: %{}` in the `Conn` struct?

And I think the assigns should be after the flash? Since it's closer to the call sight.

Reply:
And we'll also want `flash: %{}` in the foreman state and set up similar to session

https://github.com/oestrich/kalevala/blob/main/lib/kalevala/character/foreman.ex#L27

Reply:
And then in `handle_conn/2` we'll want to store the conn's flash in the foreman state struct like session, here https://github.com/oestrich/kalevala/blob/main/lib/kalevala/character/foreman.ex#L170